Firestore/Source/API/FSTUserDataReader.mm

```diff
@@ -361,22 +421,28 @@ - (void)parseSentinelFieldValue:(FIRFieldValue *)fieldValue context:(ParseContex
     context.AddToFieldTransforms(*context.path(), ServerTimestampTransform());
 
   } else if ([fieldValue isKindOfClass:[FSTArrayUnionFieldValue class]]) {
-    std::vector<FieldValue> parsedElements =
+    std::vector<google_firestore_v1_Value> parsedElements =
         [self parseArrayTransformElements:((FSTArrayUnionFieldValue *)fieldValue).elements];
-    ArrayTransform array_union(TransformOperation::Type::ArrayUnion, std::move(parsedElements));
+    ArrayTransform array_union(TransformOperation::Type::ArrayUnion,
```

Comment:
Nit: in Objective-C-land, I think this should be `arrayUnion` (and similarly `arrayRemove` below).

Reply:
Fixed both